### ISSUE TITLE
Saturated colours

### ISSUE THREAD

**Issue by quozl** [sugarlabs-appstore/aslo-v4#26]
Highly saturated colours of the default theme are difficult for neurodiverse or vision impeded students.

**Comment by srevinsaju** [comment]
![ASLO](https://user-images.githubusercontent.com/48695438/89712735-71c33c00-d99b-11ea-887f-0d63613152ac.png)

@quozl will this pallete be appropriate?

**Comment by quozl** [comment]
I don't know.  I just see five variants of blue.

**Comment by srevinsaju** [comment]
Are these five variants of blue, too saturated, or should I get a more less saturated color pallets?

**Comment by quozl** [comment]
It would depend on the context.  

https://wiki.sugarlabs.org/go/Human_Interface_Guidelines/The_Sugar_Interface/Colors gives some guidance on colours, and links to a set of colours we used in Sugar.  None of these variants of blue remind me of them.

**Comment by srevinsaju** [comment]
Fixed it :cry:

**Comment by quozl** [comment]
The activity view remains a highly saturated background;

![Screenshot_uff_2020-08-18_16:33:36](https://user-images.githubusercontent.com/2936481/90478534-988a2b00-e170-11ea-9ef4-ed4d1b0e0ac3.png)

**Comment by srevinsaju** [comment]
Oops, missed that. Should be fixed now
